### PR TITLE
Adnuntius Bid Adapter : Handle voidAuIds from adserver

### PR DESCRIPTION
**Pull request by antosarho**
Adnuntius Bid Adapter: support voidAuIds

## Type of change
- [x] Feature
- [x] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)

## Description of change
Use the information sent back via voidAuIds from the Adnuntius ad server to stop sending ad requests to inactive ad units within the Adnuntius system.

### CODE REVIEW

**Review by lksharma — CHANGES_REQUESTED** [review]
@antosarho, There are some merge conflicts within this PR. Can you address and resubmit?

**Comment by lksharma** [comment]
@antosarho, there are merge conflicts with this PR. Please resolve and push your changes.

**Review by lksharma — CHANGES_REQUESTED** [review]
@antosarho, there are merge conflicts with this PR. Please resolve and push your changes.

File: modules/adnuntiusBidAdapter.js [hunk 1]
```diff
@@ -44,6 +45,12 @@ const getSegmentsFromOrtb = function (ortb2) {
...
 //     };
 //   }
+export const misc = {
+  getUnixTimestamp: function(addDays, asMinutes) {
+    const multiplication = addDays / (asMinutes ? 1440 : 1);
+    return Date.now() + (addDays && addDays > 0 ? (1000 * 60 * 60 * 24 * multiplication) : 0);
+  }
+};
 
 //   native.sponsoredBy = ad.text.sponsoredBy?.content || '';
```
Comment (lksharma) [inline modules/adnuntiusBidAdapter.js:55]: Please remove commented code

Comment (antosarho) [reply modules/adnuntiusBidAdapter.js]: All done.

**Pull request closed without merging**